MONEY/NetworkSpentSpinTheWheelPayment.md

```diff
@@ -11,7 +11,7 @@ void _NETWORK_SPENT_SPIN_THE_WHEEL_PAYMENT(int amount, Any p1, BOOL p2, BOOL p3)
 ```
 NativeDB Introduced: v1604
 
-NativeDB Removed Parameter 3: BOOL p3
+NativeDB Removed Parameter 4: BOOL p3
```

Comment:
Just for me. What was Parameter 4 ? I don't get it on this one.

Reply:
Parameter 4 is a boolean but adding it to the function breaks C# compact iirc

Reply:
Thanks Ava!